### DISCUSSION TITLE
Telegram Bot

### DISCUSSION THREAD

**Discussion by josephfrances123** [lncrawl/lightnovel-crawler#1448] [Q&A]
Good day. I have observed that the telegram Bot hasn't been working. Is it possible to make my own bot out of this? If so, may I ask how?

**Comment by dipu-bd** [comment]
You can use the docker. Check the dev branch compose.yml : https://github.com/dipu-bd/lightnovel-crawler/blob/dev/compose.yml 

Instruction to run using docker is in the README: https://github.com/dipu-bd/lightnovel-crawler#docker

**Comment by josephfrances123** [comment]
Good day. I really appreciate your help. Thank you very much 🥰
